tests/e2e/utils/lambda_layer/powertools_layer.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import subprocess
+from pathlib import Path
+
+from checksumdir import dirhash
+
+from aws_lambda_powertools import PACKAGE_PATH
+from tests.e2e.utils.constants import CDK_OUT_PATH, SOURCE_CODE_ROOT_PATH
+from tests.e2e.utils.lambda_layer.base import BaseLocalLambdaLayer
+
+logger = logging.getLogger(__name__)
+
+
+class LocalLambdaPowertoolsLayer(BaseLocalLambdaLayer):
+    IGNORE_EXTENSIONS = ["pyc"]
+
+    def __init__(self, output_dir: Path = CDK_OUT_PATH):
+        super().__init__(output_dir)
+        self.package = f"{SOURCE_CODE_ROOT_PATH}[pydantic]"
+        self.build_args = "--platform manylinux1_x86_64 --only-binary=:all: --upgrade"
```

Comment:
I've tested this on my M1 (aarch64) mac and the E2E test ran. I guess here we're installing a dependency for the Lambda Function, so it would be fine.

In the future it would be great to have a parameter to create arm64 functions too.

Reply:
Yes, I wasn't sure on the best UX on that since it's abstracted. My initial thought is to accept an optional parameter `platform` where you could pass an enum (`x86_64`, `arm64`), and if it's `None` it'll auto-detect the platform.

What do you think?